masterlist.yaml

```diff
@@ -27191,7 +27205,6 @@ plugins:
         util: TES5Edit v3.2.2
   - name: 'imp_helm_legend.esp'
     msg:
-      - *doNotClean
```

Comment:
As I mentioned an EDID can be added by the Author to mark ITMs which should be kept.
We can't assume these don't need cleaning.